README.md

```diff
@@ -2,39 +2,27 @@
 
 # Ruby School Library / OOP
 
-> This project is in progress.
+> In this proyect we will go through the basics of OOP with Class creation, public and private methods.
```

Comment:
- [ ] [OPTIONAL] Please fix the typo you may have written by mistake, `proyect` and kindly write `project`